dispatch/modules/content/render.py

```diff
@@ -11,7 +11,10 @@ def render_node(html, node):
         if node['type'] == 'paragraph':
             return html + '<p>%s</p>' % node['data']
         else:
-            return html + embedlib.render(node['type'], node['data'])
+            try:
+                return html + embeds.render(node['type'], node['data'])
+            except EmbedException:
+                return html
```

Comment:
can this be logged somewhere (eg. as a warning into a log file or sentry)?

Reply:
Yeah, was thinking that -- meant to look into logging methods.

I think we can just use Python's built-in `logging` module and let the user configure their own handlers.